scripts/manage-members.js

```diff
@@ -0,0 +1,177 @@
+'use strict'
+
+const P = require('effd')
+const request = require('request')
+const debug = require('debug')('manage-members')
+const logger = require('../lib/logger.js')
+
+var login
+const github = require('../lib/github-client.js')
+// store/cache info about the authenticated user
+github.users.get({}).then((user) => {
```

Comment:
I'd much rather see this be requested when it's needed, rather than run immediately at script execution. I'm suspecting this is why you need to change not-related tests by manipulating `process.env.SCRIPTS`